app/api/modules.py

```diff
@@ -1,5 +1,4 @@
 from flask_rest_jsonapi import ResourceDetail
-
```

Comment:
PEP8 violation

Reply:
It Resolves pep8 / flake8 violation.

Reply:
There should be space between 3rd party and application imports

https://stackoverflow.com/a/20763446/3309666

Reply:
Resolved!
Please @iamareebjamal